scripts/reduction/reducer.py

```diff
@@ -1,4 +1,6 @@
-#pylint: disable=invalid-name
+# pylint: disable=invalid-name, protected-access, unused-variable, W0121
```

Comment:
Here I'd also say that I would be better to keep the warnings/errors that look sensible, if there's no simple way of really fixing them.